components/fieldgroups/experience-event/experienceevent-pushtracking.schema.json

```diff
@@ -56,6 +56,11 @@
               "description": "Push provider, e.g. apns or fcm",
               "meta:titleId": "experienceevent-pushtracking##xdm:pushProvider##title##90271",
               "meta:descriptionId": "experienceevent-pushtracking##xdm:pushProvider##description##23691"
+            },
+            "xdm:isLaunch": {
```

Comment:
Since "application" data type already has an "isLaunch" property and if we add another one here, it might add some confusion for the users regarding which one to use when they search for "isLaunch" properties. I would suggest if this is specific to push notifications, can we rename this to something like "isPushTriggeredLaunch" ?